src/Symfony/Component/Validator/Constraints/NoBannedWordsValidator.php

```diff
@@ -0,0 +1,118 @@
+                'a' => $this->partsToRegex(['a', '4', '/\\', '@', '^', 'aye', '∂', '/-\\', '|-\\', 'q']),
+                'b' => $this->partsToRegex(['b', '8', '6', '13', '|3', 'ß', 'P>', '|:', '!3', '(3', '/3', ')3']),
+                'c' => $this->partsToRegex(['c', '(', '¢', '<', '[', '©']),
+                'd' => $this->partsToRegex(['d', '[)', '|o', ')', 'I>', '|>', '?', 'T)', '|)', '0', '</']),
+                'e' => $this->partsToRegex(['e', '3', '&', '€', '£', 'є', 'ë', '[-', '|=-']),
+                'f' => $this->partsToRegex(['f', '|=', 'ƒ', '|#', 'ph', '/=']),
+                'g' => $this->partsToRegex(['g', '6', '&', '(_+', '9', 'C-', 'gee', '(γ,']),
+                'h' => $this->partsToRegex(['h', '#', '/-/', '[-]', ']-[', ')-(', '(-)', ':-:', '|~|', '|-|', ']~[', '}{', ']-[', '?', '}', '-{', 'hèch']),
+                'i' => $this->partsToRegex(['i', '1', '!', '|', '][', 'eye', '3y3', ']', ':']),
+                'j' => $this->partsToRegex(['j', '_|', '_/', '¿', '</', '(/', 'ʝ', ' ;']),
+                'k' => $this->partsToRegex(['k', 'X', '|<', '|{', 'ɮ', '<', '|\“']),
+                'l' => $this->partsToRegex(['l', '1', '£', '1_', 'ℓ', '|', '|_', '][_,']),
+                'm' => $this->partsToRegex(['m', '|v|', '[V]', '{V}', '|\/|', '/\/\\', '(u)', '(V)', '(\/)', '/|\\', '^^', '/|/|', '//.', '.\\', '/^^\\', '///', '|^^|']),
+                'n' => $this->partsToRegex(['n', '^/', '|V', '|\|', '/\/', '[\]', '<\>', '{\}', ']\[', '//', '^', '[]', '/V', '₪']),
+                'o' => $this->partsToRegex(['o', '0', '()', 'oh', '[]', '¤', '°', '([])']),
+                'p' => $this->partsToRegex(['p', '|*', '|o', '|º', '|^(o)', '|>', '|"', '9', '[]D', '|̊', '|7', '?', '/*', '¶', '/*', '|D']),
+                'q' => $this->partsToRegex(['q', '(_,)', '()_', '0_', '°|', '<|', '0.']),
+                'r' => $this->partsToRegex(['r', '2', '|?', '/2', '|^', 'lz', '®', '[z', '12', 'Я', '|2', 'ʁ', '|²', '.-', ',-', '|°\\']),
+                's' => $this->partsToRegex(['s', '5', '$', 'z', '§', 'ehs', 'es', '_/¯']),
+                't' => $this->partsToRegex(['t', '7', '+', '-|-', '1', '\'][\'', '†', '|²', '¯|¯']),
+                'u' => $this->partsToRegex(['u', '(_)', '|_|', 'v', 'L|', 'µ', 'J']),
+                'v' => $this->partsToRegex(['v', '\/', '1/', '|/', 'o|o']),
+                'w' => $this->partsToRegex(['w', '\/\/', 'vv', '\'//', '\\`', '\^/', '(n)', '\V/', '\X/', '\|/', '\_|_/', '\_:_/', 'Ш', 'ɰ', '`^/', '\./']),
+                'x' => $this->partsToRegex(['x', '><', 'Ж', '}{', 'ecks', '×', ')(', '8']),
+                'y' => $this->partsToRegex(['y', '7', 'j', '`/', 'Ψ', 'φ', 'λ', 'Ч', '¥', '\'/']),
+                'z' => $this->partsToRegex(['z', '≥', '2', '=/=', '7_', '~/_', ' %', '>_', '>_', '-\_', '\'/_']),
+            ]);
+        }, str_split($data)));
+    }
+
+    private function partsToRegex(array $list): string
+    {
+        $parts = array_map(function (string $word) {
```

Comment:
by using  `{...}i` instead of `/.../i` as regexp delimiters, this can be changed to 
`$parts = array_map(preg_quote(...), array_unique($list));`